designcompose/src/main/java/com/android/designcompose/VariableManager.kt

```diff
@@ -200,23 +205,25 @@ internal object VariableManager {
     }
 
     // Given a variable ID, return the color associated with it
-    internal fun getColor(varId: String, variableState: VariableState): Color? {
+    internal fun getColor(varId: String, fallback: Color?, variableState: VariableState): Color? {
         // Resolve varId into a variable. If a different collection has been set, this will return
         // a variable of the same name from the override collection.
         val variable = resolveVariable(varId, variableState)
         variable?.let { v ->
             return v.getColor(varMap, variableState)
         }
-        return null
+        Feedback.documentVariableMissingWarning(currentDocId, varId)
+        return fallback
     }
 
     // Given a variable ID, return the number associated with it
-    internal fun getNumber(varId: String, variableState: VariableState): Float? {
+    internal fun getNumber(varId: String, fallback: Float?, variableState: VariableState): Float? {
```

Comment:
How does fallback work? Does it get the fallback value from the doc?

Reply:
Yes. The fallback is the value, the number in this case.